google/cloud/status.h

```diff
@@ -102,7 +105,9 @@ class Status {
 };
 
 inline std::ostream& operator<<(std::ostream& os, Status const& rhs) {
-  return os << rhs.error_message() << " [" << rhs.status_code()
+  return os << rhs.error_message()
+            << StatusCodeToString(static_cast<StatusCode>(rhs.status_code()))
```

Comment:
Just stream the StatusCode?  It has its own operator<<().

Reply:
Much better, thanks.